packages/loader/container-loader/src/container.ts

```diff
@@ -842,6 +842,16 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         try {
             // Ensure that we raise all key events even if one of these throws
             try {
+                // Raise event first, to ensure we capture _lifecycleState before transition.
+                // This gives us a chance to know what errors happened on open vs. on fully loaded container.
+                this.mc.logger.sendTelemetryEvent(
+                    {
+                        eventName: "ContainerDispose",
```

Comment:
@vladsud - what do you think - "ContainerDispose", or "ContainerClose" with a prop saying it's dispose?  Or "ContainerClose_Dispose" :P

Reply:
> "ContainerClose" with a prop saying it's dispose

I vote for this.  Whether someone calls close or dispose should mostly be abstracted from us when we look at telemetry, it mostly means the same thing, and if we need to search out the distinction we'll have the easy way to query it.

Reply:
Btw - The idea with "ContainerClose_Dispose" is we can update all our queries to `Data_eventName has "ContainerClose"` but the fact that it's Dispose is in your face right in the event name.  It's a little obscure/particular so probably not a good idea in practice.